test/integration/toolchains/gnu/test_gnutoolchain.py

```diff
@@ -436,30 +436,21 @@ def test_conf_build_does_not_exist():
     host = textwrap.dedent("""
     [settings]
     arch=x86_64
-    build_type=Release
-    compiler=gcc
-    compiler.cppstd=gnu17
-    compiler.libcxx=libstdc++11
-    compiler.version=13
     os=Linux
-    [conf]
-    tools.build:compiler_executables={'c': 'x86_64-linux-gnu-gcc', 'cpp': 'x86_64-linux-gnu-g++'}
```

Comment:
Let's keep the conf here to ensure that the `CC_FOR_BUILD` remains correct. For instance:

```ini
[conf]
    tools.build:compiler_executables={'c': '/usr/bin/gcc', 'cpp': '/usr/bin/g++'}
```